.ado/pull-request.yml

```diff
@@ -11,89 +11,92 @@ pr:
 pool:
   vmImage: 'windows-2019'
 
+variables:
+  System.Debug: true
+
 jobs:
-  - job: V8JsiBuild
-    timeoutInMinutes: 180
-    displayName: Build v8jsi.dll
-    strategy:
-      matrix:
-        Desktop|x64|Debug:
-          BuildConfiguration: Debug
-          BuildPlatform: x64
-          AppPlatform: win32
-        Desktop|x86|Debug:
-          BuildConfiguration: Debug
-          BuildPlatform: x86
-          AppPlatform: win32
-        Desktop|ARM64|Debug:
-          BuildConfiguration: Debug
-          BuildPlatform: arm64
-          AppPlatform: win32
-        Desktop|x64|Release:
-          BuildConfiguration: Release
-          BuildPlatform: x64
-          AppPlatform: win32
-        Desktop|x86|Release:
-          BuildConfiguration: Release
-          BuildPlatform: x86
-          AppPlatform: win32
-        Desktop|ARM64|Release:
-          BuildConfiguration: Release
-          BuildPlatform: arm64
-          AppPlatform: win32
-        Universal|x64|Debug:
-          BuildConfiguration: Debug
-          BuildPlatform: x64
-          AppPlatform: uwp
-        Universal|x86|Debug:
-          BuildConfiguration: Debug
-          BuildPlatform: x86
-          AppPlatform: uwp
-        Universal|ARM64|Debug:
-          BuildConfiguration: Debug
-          BuildPlatform: arm64
-          AppPlatform: uwp
-        Universal|x64|Release:
-          BuildConfiguration: Release
-          BuildPlatform: x64
-          AppPlatform: uwp
-        Universal|x86|Release:
-          BuildConfiguration: Release
-          BuildPlatform: x86
-          AppPlatform: uwp
-        Universal|ARM64|Release:
-          BuildConfiguration: Release
-          BuildPlatform: arm64
-          AppPlatform: uwp
+  # - job: V8JsiBuild

```

Comment:
># [](start = 2, length = 1)

You probably want to uncomment these

Reply:
Done. I actually published the raw change so I could use GitHub's diff to track down these changes.

Latest update should be clean and ready to merge.